ironfish-rust/src/keys/view_keys.rs

```diff
@@ -92,6 +92,51 @@ impl IncomingViewKey {
     }
 }
 
+#[derive(Clone)]
+pub struct ViewKey {
+    pub authorizing_key: jubjub::SubgroupPoint,
+    pub nullifier_deriving_key: jubjub::SubgroupPoint,
+}
+
+impl ViewKey {
+    /// Load a key from a string of hexadecimal digits
+    pub fn from_hex(value: &str) -> Result<Self, IronfishError> {
+        match hex_to_bytes(value) {
+            Err(_) => Err(IronfishError::InvalidViewKey),
+            Ok(bytes) => {
+                if bytes.len() != 64 {
+                    Err(IronfishError::InvalidViewKey)
+                } else {
+                    let mut authorizing_key_bytes = [0; 32];
+                    let mut nullifier_deriving_key_bytes = [0; 32];
+                    authorizing_key_bytes.clone_from_slice(&bytes[..32]);
+                    nullifier_deriving_key_bytes.clone_from_slice(&bytes[32..]);
+                    let authorizing_key =
```

Comment:
Not stoked about these `unwrap`s, but the code will get way uglier if we try to deal with the underlying types of `CtOption`. I was thinking of extending the CtOption in our codebase with something like
```
impl From<CtOption<T>> for Result<T, IronfishError> {
...
```

But we don't include the `subtle` library dependency, so not sure if we can do that without explicitly adding the dependency.